### PR TITLE
Add FreeBSD support, using telegraf as installed by pkg(8)

### PR DESCRIPTION
**Pull request by rvstaveren**
#### Pull Request (PR) description

This adds support for FreeBSD, using telegraf as installed by pkg(8). 
Where applicable, spec logic from Darwin support was used. 
FreeBSD's telegraf is in the package collection and needs no repository management.

#### This Pull Request (PR) fixes the following issues
n/a

### CODE REVIEW

**Comment by bastelfreak** [comment]
please also add freebsd to https://github.com/voxpupuli/puppet-telegraf/blob/master/metadata.json

File: metadata.json [hunk 1]
```diff
@@ -74,6 +74,13 @@
       "operatingsystemrelease": [
         "18"
       ]
+    },
+    {
+      "operatingsystem": "FreeBSD,
```
Comment (bastelfreak) [inline metadata.json]: please add the missing `"` and fix the alignment in the lines below (I highly recommend using soft tabs)

Comment (rvstaveren) [reply metadata.json]: Hi, my bad. The editor decided to insert some quotes by itself which I overlooked. Should be ok now. Hard tabs are gone too

**Comment by rvstaveren** [comment]
Ok this seems to trigger additional errors, I’m looking into this

**Pull request merged**